src/lib.rs

```diff
@@ -428,6 +428,22 @@ impl Build {
         self
     }
 
+    /// removes a flag from self.flags that was added by Build.flag()
```

Comment:
Please don't mention `self.flags` since it is an implementation details, and also it's better to use [`Build::flag`] since it will create a url in the doc generated:

```suggestion
    /// removes a compiler flag that was added by [`Build::flag`]
```